src/main/java/frc/robot/subsystems/shooter/Shooter.java

```diff
@@ -0,0 +1,87 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems.shooter;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+/**
+ * Add your docs here.
+ */
+public class Shooter {
+
+    private final int leftMotorCanId = 11;
+    private final int rightMotorCanId = 12;
+    private final CANSparkMax leftMotor = new CANSparkMax(leftMotorCanId, MotorType.kBrushless);
+    private final CANSparkMax rightMotor = new CANSparkMax(rightMotorCanId,MotorType.kBrushless);
+
+    private final double wheelDiameter = 4; //inch
+    private final double Gravity = 386.0886;
+    private final double maxRpm = 7640.0; 
+    private final double launcherHeight = 24.0; //inch
+    private final double energyLost = 0.66; 
+    private final double wheelCircumference = wheelDiameter*Math.PI;
+    private final double maxTangentialSpeed = ( wheelCircumference * (maxRpm /60));
+    private final double initialUpperBound = 1.0; //%100 speed
+    private final double initialLowerBound = 0.0;//%0 speed
```

Comment:
perhaps `INITIAL_UPPER_BOUND_SPEED`? initialUpperBound and initialLowerBound aren't very descriptive